run/test/specs/utils/open_app.ts

```diff
@@ -266,6 +266,7 @@ const openAndroidApp = async (
     ignoreUnimportantViews: false,
     allowInvisibleElements: true,
     enableMultiWindows: true,
+    disableIdLocatorAutocompletion: true,
```

Comment:
is this helping?

Reply:
yeah that is included for the new testTag on Android